src/components/ReportActionItem/ReportPreview.tsx

```diff
@@ -226,12 +225,15 @@ function ReportPreview({
     const shouldShowSubtitle = !isScanning && (shouldShowSingleRequestMerchantOrDescription || numberOfRequests > 1);
 
     const {supportText} = useMemo(() => {
-        if (formattedMerchant) {
+        if (formattedMerchant && formattedMerchant !== CONST.TRANSACTION.DEFAULT_MERCHANT) {
             return {supportText: formattedMerchant};
         }
         if (formattedDescription ?? moneyRequestComment) {
             return {supportText: formattedDescription ?? moneyRequestComment};
         }
+        if (formattedMerchant === CONST.TRANSACTION.DEFAULT_MERCHANT) {
```

Comment:
```suggestion
        if (formattedMerchant === CONST.TRANSACTION.DEFAULT_MERCHANT || formattedMerchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT) {
```

Reply:
I am not sure about this change, if the merchant is  `(none)` I dont think we want to show that